docker/Dockerfile

```diff
@@ -0,0 +1,41 @@
+FROM node:18-bullseye-slim
+
+RUN DEBIAN_FRONTEND=noninteractive \
+  && apt-get update \
+  && apt-get upgrade -yqq \
+  && apt-get install --no-install-recommends -y \
+    build-essential \
+    curl \
+    ca-certificates \
+    llvm-13 \
+    llvm-13-dev \
+    libclang-common-13-dev \
+    zlib1g-dev \
+  && apt-get clean \
+  && rm -fr /var/lib/apt/lists/*
+
+ENV PATH="/usr/lib/llvm-13/bin/:${PATH}"
+
+# Install Rust
+RUN curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | bash -s -- -y
+ENV PATH="/root/.cargo/bin:${PATH}"
+RUN rustup target add wasm32-unknown-unknown
```

Comment:
is also probably a good idea to add wasm32-wasi, we may use this for running certain tests and we use the wasi target for debugging pourpouses

Reply:
Ok